modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Chart.js

```diff
@@ -139,14 +146,24 @@ export default function Chart({
 
 	const [, addHistoricalWarning] = useHistoricalWarning();
 
-	const {actions, state: chartState} = useChartState();
+	const chartState = useChartState();
+
+	const addDataSetItems = useAddDataSetItems();
+
+	const setLoading = useSetLoading();
+
+	const changeTimeSpanKey = useChangeTimeSpanKey();
+
+	const previousTimeSpan = usePreviousTimeSpan();
+
+	const nextTimeSpan = useNextTimeSpan();
```

Comment:
why these new lines between declarations? (this is a really useful comment)

Reply:
Because [Brian wants it that way](https://github.com/liferay/liferay-frontend-projects/issues/15).

Reply:
:open_mouth: I thought that was only if they are not sorted

Reply:
Yes, that is the reason! As Greg suggested in his comment, I'm going to reorder alphabetically these declarations as much as possible :)